### PR TITLE
Added the ansible installer for dashboard, keystone and SRM toolchain

### LINKED ISSUE #675: Create a separate installer for SODA Dashboard and all its dependencies
**Issue/Feature Description:**
As part of the SODA project installation overhaul, the project installations are being built separately for all the SODA Framework projects.
The SODA Dashboard also must be able to be installed using an independent installer and also install all the dependencies needed for the Dashboard to operate properly.

- [ ] SODA Dashboard
- [ ] SODA Authentication using Keystone. (Required for Strato and Terra projects)
- [ ] SRM Toolchain with Prometheus, Grafana and Alertmanager. (Required for Delfin visualization)

**Why this issue to fixed / feature is needed(give scenarios or use cases):**
The SODA project installers are being separated to enable users to install those framework projects that they wish to use and test. The previous installer made it mandatory for the user to install Keystone and dashboard irrespective of whether the project required auth.
This installer should enable the user to install the Dashboard, Keystone and SRM toolchain independently and use it along with any of the framework projects installed.
**How to reproduce, in case of a bug:**

**Other Notes /  Environment Information: (Please give the env information, log link or any useful information for this issue)**

### PR DESCRIPTION
**Pull request by anvithks**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
/kind enhancement

**What this PR does / why we need it**:
This PR adds the ansible installer that allows the user to install the SODA Experience independently.

- [x] SODA Authentication using Keystone
- [x] SODA Dashboard 
- [x] SRM Toolchain required for Delfin visualization.

**Which issue(s) this PR fixes**:
Fixes #675 

**Test Report Added?**:
/kind TESTED

**Test Report**:
Tested on Ubuntu 18.04 and 20.04

**Special notes for your reviewer**:

### CODE REVIEW

**Review by joseph-v — APPROVED** [review]
LGTM

File: install/ansible/clean.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+# Defines some clean processes when banishing the cluster.
+
+- name: destory an opensds cluster
```
Comment (skdwriting) [inline install/ansible/clean.yml:18]: do we call it as opensds cluster??!

Comment (skdwriting) [reply install/ansible/clean.yml:18]: Why this is part of dashboard cleanup?

File: install/ansible/group_vars/auth.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+# file as a good configuration file when no variable in it.
+dummy:
+
+
+###########
+# GENERAL #
+###########
+
+# OpenSDS authentication strategy, support 'noauth' and 'keystone'.
+opensds_auth_strategy: keystone
```
Comment (skdwriting) [inline install/ansible/group_vars/auth.yml:26]: should it not be part of soda installer?

File: install/ansible/group_vars/common.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+# GENERAL #
+###########
+
+# This field indicates local machine host ip
+# For HA configuration, add the IP on which the Core Services (Hotpot) are reachable from the Kubernetes node.
+host_ip: 127.0.0.1
+
+# These fields below will specify the tag based on install_from type
+repo_branch: master
+release_version: v1.7.1
```
Comment (skdwriting) [inline install/ansible/group_vars/common.yml:31]: why this here?

File: install/ansible/group_vars/dashboard.yml [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+dashboard_docker_image: sodafoundation/dashboard:{{ dashboard_release_version }}
+
+##############
+# SOURCECODE #
+##############
+
+# If user specifies intalling from source code, then he can choose the specific
+# repository branch
+dashboard_repo_branch: "{{ repo_branch }}"
```
Comment (skdwriting) [inline install/ansible/group_vars/dashboard.yml:52]: Should it not come from the SODA installer configurations? Do we need all these project configurations as a part of dashboard? In that case, we need to have project enable / disable support right?!
I am thinking, should we have a simple installer for dashboard and rest handled by SODA installer?!

**Pull request merged**